packages/phone-sensors/README.md

```diff
@@ -23,6 +23,41 @@ One group uses a standard data collection service, but the other one uses a spec
 to label the collected data with the most accurate timestamp.
 The collected data from the sensors, will be a [TriAxial](#triaxial) record, described below.
 
+### Setup (Optional)
+
+This plugin can be optionally registered using its loader during the framework initialization:
+
+```ts
+// ... platform imports
+import { awarns } from '@awarns/core';
+import { demoTasks } from '../tasks';
+import { demoTaskGraph } from '../graph';
+import { registerPhoneSensorsPlugin } from '@awarns/phone-sensors';
+
+awarns
+  .init(
+    demoTasks,
+    demoTaskGraph,
+    [
+      registerPhoneSensorsPlugin(false)
```

Comment:
How about using a settings object instead?

This reads a bit strange when using it as a public API. You don't know what this false is for unless you go to the docs or the function definition.

What do you think?